src/Microsoft.DotNet.Cli.Utils/FixedPathCommandFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class FixedPathCommandFactory : ICommandFactory
```

Comment:
I thought fixed, because every command created by it, will have its paths set based on the parameters passed to the factory through the constructor.